pyfr/polys.py

```diff
@@ -165,14 +162,14 @@ def degrees(self):
 class QuadPolyBasis(BasePolyBasis):
     name = 'quad'
 
-    def ortho_basis_at_py(self, p, q):
+    def _ortho_basis_at(self, p, q):
```

Comment:
Some comments on what p, q and r are would be good. It is not immediately clear what they are.